flow-typed/Collections.js

```diff
@@ -14,14 +14,21 @@ declare type CollectionState = {
   pending: CollectionGroup,
   edited: CollectionGroup,
   builtin: CollectionGroup,
-  saved: Array<string>,
   isResolvingCollectionById: { [string]: boolean },
   error?: string | null,
+  queue: Collection,
 };
```

Comment:
- [ ] List cannot be opened in legacy apps?

This may or may not be an issue, but just wanted to be sure:

- If I create a list using this branch, I can see the list from production, but I cannot access it.  The url ends up being `https://odysee.com/?lid=adaadeff-88e1-485b-ab8c-62d5f037bfa1`

I understand there are URL changes ("list" to "playlist"), but from a data perspective, the app should just retrieve the ID and display it, right?  Trying to figure out why it would affect production or legacy apps -- how the app routes it shouldn't be impacted.